src/standard_name.jl

```diff
@@ -0,0 +1,208 @@
+    "land~industry__gross_water_demand_volume_flux" =>
+        @optic(_.land.demand.industry.demand.demand_gross),
+    "land~industry__net_water_demand_volume_flux" =>
+        @optic(_.land.demand.industry.demand.demand_net),
+    "land~livestock__gross_water_demand_volume_flux" =>
+        @optic(_.land.demand.livestock.demand.demand_gross),
+    "land~livestock__net_water_demand_volume_flux" =>
+        @optic(_.land.demand.livestock.demand.demand_net),
+    "land~irrigated-paddy__irrigation_trigger_flag" =>
+        @optic(_.land.demand.paddy.parameters.irrigation_trigger),
+    "land~irrigated-paddy__gross_water_demand_volume_flux" =>
+        @optic(_.land.demand.paddy.variables.demand_gross),
+    "land~irrigated-non-paddy__irrigation_trigger_flag" =>
+        @optic(_.land.demand.nonpaddy.parameters.irrigation_trigger),
+    "land~irrigated-non-paddy__gross_water_demand_volume_flux" =>
+        @optic(_.land.demand.nonpaddy.variables.demand_gross),
+    "land~irrigated__allocated_water_volume_flux" =>
+        @optic(_.land.allocation.variables.irri_alloc),
+    "land~non-irrigated__allocated_water_volume_flux" =>
+        @optic(_.land.allocation.variables.nonirri_alloc),
+    "subsurface_water_abstraction__volume_flux" =>
+        @optic(_.land.allocation.variables.act_groundwater_abst),
+    "subsurface_water__hydraulic_head" =>
+        @optic(_.routing.subsurface_flow.aquifer.variables.head),
+    "subsurface_water_sat-zone_top__net_recharge_volume_flux" =>
+        @optic(_.routing.subsurface_flow.boundaries.recharge.variables.flux),
+    "land_drain_water~to-subsurface__volume_flow_rate" =>
+        @optic(_.routing.subsurface_flow.boundaries.drain.variables.flux),
+    "river_water~to-subsurface__volume_flow_rate" =>
+        @optic(_.routing.subsurface_flow.boundaries.river.variables.flux),
+)
+
+const sediment_standard_name_map = Dict{String, ComposedFunction}(
+    "atmosphere_water__precipitation_volume_flux" =>
+        @optic(_.land.atmospheric_forcing.precipitation),
+    "river_water__volume_flow_rate" =>
+        @optic(_.routing.river_flow.hydrological_forcing.q_river),
+    "river_water__depth" =>
+        @optic(_.routing.river_flow.hydrological_forcing.waterlevel_river),
+    "land_surface_water__volume_flow_rate" =>
+        @optic(_.land.hydrological_forcing.q_land),
+    "land_surface_water__depth" => @optic(_.land.hydrological_forcing.waterlevel_land),
+    "vegetation_canopy_water__interception_volume_flux" =>
+        @optic(_.land.hydrological_forcing.interception),
+    "soil_erosion~rainfall__mass_flow_rate" =>
+        @optic(_.land.rainfall_erosion.variables.amount),
+    "soil_erosion~overland_flow__mass_flow_rate" =>
+        @optic(_.land.overland_flow_erosion.variables.amount),
+    "soil_erosion__mass_flow_rate" => @optic(_.land.soil_erosion.variables.amount),
+    "soil_erosion_clay__mass_flow_rate" => @optic(_.land.soil_erosion.variables.clay),
```

Comment:
This one is needed for all particle class clay/silt/sand/sagg/lagg
```suggestion
    "soil_erosion_clay__mass_flow_rate" => @optic(_.land.soil_erosion.variables.clay),
    "soil_erosion_silt__mass_flow_rate" => @optic(_.land.soil_erosion.variables.silt),
    "soil_erosion_sand__mass_flow_rate" => @optic(_.land.soil_erosion.variables.sand),
    "soil_erosion_aggregates~small__mass_flow_rate" => @optic(_.land.soil_erosion.variables.sagg),
    "soil_erosion_aggregates~large__mass_flow_rate" => @optic(_.land.soil_erosion.variables.lagg),
```

Reply:
See previous comment about standard use of the sediment model.